docs/reference/03-filters.md

```diff
@@ -0,0 +1,67 @@
+# Query filters
+
+## Type filters
+
+When querying for data, you often need to query for specific things, such as if a database column contains a word,
```

Comment:
"When querying for data, you often need to query for specific things"

This sentence offers no information... 😅 Maybe remove or rework.

Reply:
> You probably want to build detailed queries, such as if a database column contains a word, [...]

?

Reply:
On this page, we explore query filters. Query filters help us ...

... All of these queries are fully type-safe and independent of the underlying database.